index.d.ts

```diff
@@ -24,7 +24,7 @@ declare namespace Eris {
   // TODO there's also toJSON(): JSONCache, though, SimpleJSON should suffice
 
   type GuildTextableChannel = TextChannel | NewsChannel
-  type TextableChannel = Textable & GuildTextableChannel | PrivateChannel;
+  type TextableChannel = Textable & (GuildTextableChannel | PrivateChannel);
```

Comment:
```suggestion
  type TextableChannel = (GuildTextable & GuildTextableChannel) | (Textable & PrivateChannel)
```
not on my laptop rn so this is the easiest way to change it